code/modules/mob/logout.dm

```diff
@@ -7,14 +7,18 @@
 	create_log(MISC_LOG, "Logged out")
 	// `holder` is nil'd out by now, so we check the `admin_datums` array directly
 	//Only report this stuff if we are currently playing.
-	if(GLOB.admin_datums[ckey] && SSticker && SSticker.current_state == GAME_STATE_PLAYING)
+	if(GLOB.admin_datums[ckey] && SSticker.current_state == GAME_STATE_PLAYING)
 		var/datum/admins/temp_admin = GLOB.admin_datums[ckey]
-		// Triggers on people with banhammer power only - no mentors tripping the alarm
-		if(temp_admin.rights & R_BAN)
+		if(temp_admin.rights & R_MENTOR)
```

Comment:
Not sure how the permissions are set up. But what if an admin/host with `R_MENTOR` in his permission set logs out. It will only call the mentor has logged out part. And not the admin part.
Prob want to switch these two

Reply:
Explained in discord. This is fine and works in production.